src/wallet/rpcwallet.cpp

```diff
@@ -200,7 +200,9 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
     ret.pushKV("isshield", pTransDest == nullptr);
 
     if (pTransDest) {
-        ret.pushKV("scriptPubKey", HexStr(GetScriptForDestination(*pTransDest)));
+        CScript scriptPubKey = GetScriptForDestination(*pTransDest);
+        ret.pushKV("scriptPubKey", HexStr(scriptPubKey));
+        ret.pushKV("ischange", pwallet->IsChange(scriptPubKey));
```

Comment:
"ischange" should be added in the help too.

Reply:
true, updated.